### PR TITLE
Feature/improve dashboard view

### LINKED ISSUE #371: Improve dashboard layout
Some users reported that the current layout is confusing, new suggestion:

Bildschirmfoto 2020-12-02 um 14 59 07

### PR DESCRIPTION
**Pull request by jagaSto**
**What changes does this PR introduce**

* fixes #371 - adds better visual separation between ask-for-help request of a user and notifications

_Briefly describe your changes here_

* adds visual separation, similar to the one on landing page
* removes `react-web-tabs` dependency 
* aligns toogle for open|solved requests with toogle on main page

_Screenshots_

Mobile:

Screenshot 2020-12-05 at 18 21 49

Desktop:

Screenshot 2020-12-07 at 22 10 51

### CODE REVIEW

File: src/views/Dashboard.jsx [hunk 1]
```diff
@@ -143,41 +142,71 @@ function Dashboard(props) {
...
+                .
+              </div>
+            )
+            : offers.map((offer) => <Notification location={offer.location} id={offer.id} key={offer.id} />)
+        }
+      </div>
 
       <div className="mt-12">
-        <DeleteAccountButton className="rounded text-white p-3 btn-main bg-primary hover:opacity-75 float-right" user={user} />
+        <DeleteAccountButton className="rounded text-white p-3  mr-4 btn-main bg-primary hover:opacity-75 float-right" user={user} />
```
Comment (mauriceackel) [inline src/views/Dashboard.jsx:209]: Why did you add the margin? I kinda liked it when everything was aligned on the same line.

File: src/views/Dashboard.jsx [hunk 2]
```diff
@@ -143,41 +142,71 @@ function Dashboard(props) {
...
+          <button
+            type="button"
+            onClick={() => {
+              setIsOpenEntriesView(false);
+            }}
+            className={`text-white items-center rounded-r px-1 py-1 xs:px-6 md:py-3 btn-main
+             ${isOpenEntriesView ? 'btn-light-green' : 'btn-dark-green'} hover:opacity-75`}
+          >
+            {t('views.dashboard.tabs.solved')}
+          </button>
```
Comment (mauriceackel) [inline src/views/Dashboard.jsx:174]: In my opinion, the whole tab selection takes up a bit too much space on desktop. Maybe we can reduce the margins and paddings on desktop, somewhat similar to the mobile layout? Maybe even more?

**Review by mauriceackel — APPROVED** [review]
From a technical standpoint, LGTM. I added some minor comments regarding the design.

File: src/views/Dashboard.jsx (hunk 1 above)
Comment (jagaSto) [reply src/views/Dashboard.jsx:209]: because otherwise, the button looks crappy on mobile (it would stick completely to the side like the gray background for the sections) --> I will make sure to remove it on desktop

File: src/views/Dashboard.jsx (hunk 2 above)
Comment (jagaSto) [reply src/views/Dashboard.jsx:174]: sure, let me take a look
